test/unittest/unittest_nntrainer_appcontext.cpp

```diff
@@ -94,7 +94,7 @@ class CustomOptimizer : public nntrainer::Optimizer {
   /** Full custom optimizer example which overrides all functions */
   const std::string getType() const { return "identity_optimizer"; }
 
-  float getLearningRate() { return 1.0f; }
+  float getLearningRate() const { return 1.0f; }
```

Comment:
Could you put `override` here?
```suggestion
  float getLearningRate() const override { return 1.0f; }
```

Reply:
and for other members as well?